src/importlinter/domain/helpers.py

```diff
@@ -99,14 +98,8 @@ def module_expression_to_modules(graph: ImportGraph, expression: ModuleExpressio
     if not expression.has_wildcard_expression():
```

Comment:
General question - do we consider the functions within `helpers.py` and `contract_utils.py` to be part of the import-linter public API? I.e. do we have to consider very carefully breaking changes to these functions? I suppose so, since they are likely used by custom contracts.